### DISCUSSION TITLE
Missing batch function for guided tours

### DISCUSSION THREAD

**Discussion by hytsch** [joomla/joomla-cms#41143] [2. Cold Features]
### Steps to reproduce the problem
Open the Guided Tours; mark several tours or steps, try to set the access level to registered using the batch function. 

### Expected result
The batch is available for this

### Actual result
The batch function is missing

### System information (as far as possible)
j 3.4.2

### Additional comments
Because the same thing should always work the same way, as an admin I expect the batch function to be available for Guided Tours as well.

**Comment by brianteeman** [comment]
seriously?

**Comment by hytsch** [comment]
![11-07-2023_08-32-59](https://github.com/joomla/joomla-cms/assets/4489029/e9e28b20-a450-45dc-9958-272cda14989b)

**Comment by brianteeman** [comment]
I understood what you meant I just dont see why everything has to be replicated everywhere.
Examples
Modules has a duplicate action
Plugins has no actions at all
etc etc
